src/app/app.module.ts

```diff
@@ -35,6 +37,7 @@ import { AmComponentComponent } from './am-component/am-component.component';
     TjsMultiplicationComponent,
     RcRandomNumberComponent,
     AmComponentComponent
+    TkAdditionComponent
```

Comment:
Seems you're missing a comma in your merge conflict resolution... Can you fix this and I'll be able to accept your PR...

Reply:
Alright, it should be fixed now.